otelopscol/Makefile

```diff
@@ -123,9 +123,9 @@ $(COLLECTOR_BINARY_NAME): ocb-generate
 .PHONY: collector-package-list
 collector-package-list: ocb-generate
 	cd $(COLLECTOR_BUILD_TEMP_DIR) && \
-		GOWORK=off go list -deps -f '{{ .ImportPath }} {{ .Module }}' ./... > ../packages.txt
-	grep -v '<nil>' packages.txt | sort -u > packages.txt
-	rm -rf $(COLLECTOR_BUILD_TEMP_DIR)
+		GOWORK=off go list -deps -f '{{ .ImportPath }} {{ .Module }}' ./... |\
+		grep -v '<nil>' |\
+		sort -u > ../packages.txt
```

Comment:
Is it possible to move `collector-package-list` (and possibly other targets) to a top-level `Makefile.common` that could be include here ?

I've recently found that `opentelemetry-collector-contrib` does this in every component which is very convenient.

Reply:
Anything that is within the generated distribution folder has to come from the `distrogen` template in `cmd/distrogen/templates`. Any of these templates are designed to be completely standalone from where they're generated, so they won't share any code.

I do use [reusable Make code](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/tree/master/make) that is [imported by our components](https://github.com/GoogleCloudPlatform/opentelemetry-operations-collector/blob/master/components/google-built-opentelemetry-collector/Makefile) since the components in our repo are specific to here.

Reply:
Thanks for the explanation!